docker-compose.yml

```diff
@@ -79,6 +85,8 @@ services:
       - s3-init
     container_name: uc-historic-data-importer
     command: >-
+      --logging.level.root=DEBUG
+      --spring.main.banner-mode=off
```

Comment:
If this is the docker compose for prod, is there a reason to have this at DEBUG level?

Reply:
It's not, it's local dev testing only